pontoon/contributors/views.py

```diff
@@ -32,11 +32,6 @@
 
 
 @login_required(redirect_field_name='', login_url='/403')
```

Comment:
You have to remove this line as well, otherwise this [decorator](https://docs.djangoproject.com/en/2.2/topics/http/decorators/) will make the `contributor_email` view (which handles http://localhost:8000/contributors/your.name@example.com/ for example) only accessible to logged in users.